--- libguestfs-1.52.1/m4/guestfs-appliance.m4.orig	2024-06-18 06:02:35.802278052 +0200
+++ libguestfs-1.52.1/m4/guestfs-appliance.m4	2024-06-18 19:53:54.552061673 +0200
@@ -130,7 +130,7 @@ if test "x$ENABLE_APPLIANCE" = "xyes"; t
     AC_SUBST([DISTRO])
 fi
 AM_CONDITIONAL([HAVE_RPM],
-    [AS_CASE([$DISTRO], [REDHAT | SUSE | OPENMANDRIVA | MAGEIA | OPENCLOUDOS ], [true],
+    [AS_CASE([$DISTRO], [REDHAT | SUSE | OPENMANDRIVA | MAGEIA | OPENCLOUDOS | PLD ], [true],
                         [*], [false])])
 AM_CONDITIONAL([HAVE_DPKG],
     [AS_CASE([$DISTRO], [DEBIAN | UBUNTU ], [true],
--- libguestfs-1.52.1/appliance/packagelist.in.orig	2024-05-13 15:12:19.002830902 +0200
+++ libguestfs-1.52.1/appliance/packagelist.in	2024-06-18 06:02:35.802278052 +0200
@@ -238,6 +238,27 @@ ifelse(OPENCLOUDOS,1,
   zstd
 )
 
+ifelse(PLD,1,
+  cryptsetup
+  dhcp-client
+  grub2
+  iproute2
+  iputils
+  kernel
+  libldm
+  nilfs-utils
+  ntfsprogs
+  ntfs-3g
+  openssh-clients
+  policycoreutils
+  reiserfsprogs
+  rpm-lib
+  systemd              dnl for /sbin/reboot and udevd
+  vim-light
+  xz
+  zstd
+)
+
 include(guestfsd.deps)
 
 acl
