diff -urNp -x '*.orig' libguestfs-1.40.2.org/daemon/Makefile.am libguestfs-1.40.2/daemon/Makefile.am
--- libguestfs-1.40.2.org/daemon/Makefile.am	2019-02-07 15:45:56.644955783 +0100
+++ libguestfs-1.40.2/daemon/Makefile.am	2021-03-30 23:01:01.769494093 +0200
@@ -222,6 +222,7 @@ guestfsd_LDADD = \
 	$(JANSSON_LIBS) \
 	$(SELINUX_LIBS) \
 	$(AUGEAS_LIBS) \
+	$(LIBXML2_LIBS) \
 	$(HIVEX_LIBS) \
 	$(SD_JOURNAL_LIBS) \
 	$(top_builddir)/gnulib/lib/.libs/libgnu.a \
@@ -255,6 +256,7 @@ guestfsd_CFLAGS = \
 	$(WARN_CFLAGS) $(WERROR_CFLAGS) \
 	$(RPC_CFLAGS) \
 	$(AUGEAS_CFLAGS) \
+	$(LIBXML2_CFLAGS) \
 	$(HIVEX_CFLAGS) \
 	$(SD_JOURNAL_CFLAGS) \
 	$(JANSSON_CFLAGS) \
