--- libguestfs-1.20.1/src/Makefile.am.orig	2012-12-20 16:40:46.000000000 +0100
+++ libguestfs-1.20.1/src/Makefile.am	2013-01-06 19:21:18.854294111 +0100
@@ -178,6 +178,7 @@
 	$(PCRE_LIBS) $(MAGIC_LIBS) \
 	$(LIBVIRT_LIBS) $(LIBXML2_LIBS) \
 	$(YAJL_LIBS) \
+	$(SELINUX_LIB) \
 	../gnulib/lib/libgnu.la \
 	$(GETADDRINFO_LIB) \
 	$(HOSTENT_LIB) \
--- libguestfs-1.24.0/gobject/Makefile.am.orig	2013-10-19 09:15:17.000000000 +0200
+++ libguestfs-1.24.0/gobject/Makefile.am	2013-10-26 07:35:22.919512265 +0200
@@ -54,8 +54,8 @@
 libguestfs_gobject_1_0_la_CFLAGS = -I$(top_srcdir)/src -I$(srcdir)/include \
                                    -DGUESTFS_PRIVATE=1 \
                                    $(GOBJECT_CFLAGS)
-libguestfs_gobject_1_0_la_LDFLAGS = $(LDFLAGS) -L$(top_builddir)/src
-libguestfs_gobject_1_0_la_LIBADD = -lguestfs $(GOBJECT_LIBS) $(GIO_LIBS)
+libguestfs_gobject_1_0_la_LDFLAGS = $(LDFLAGS)
+libguestfs_gobject_1_0_la_LIBADD = $(top_builddir)/src/libguestfs.la $(GOBJECT_LIBS) $(GIO_LIBS)
 
 pkgconfigdir = $(libdir)/pkgconfig
 pkgconfig_DATA = libguestfs-gobject-1.0.pc
@@ -80,7 +80,7 @@
 Guestfs-1.0.gir: $(libname)
 Guestfs_1_0_gir_INCLUDES = GObject-2.0 Gio-2.0
 Guestfs_1_0_gir_CFLAGS = $(INCLUDES) -I$(srcdir)/include
-Guestfs_1_0_gir_LIBS = $(libname)
+Guestfs_1_0_gir_LIBS = $(libname) $(top_builddir)/src/libguestfs.la
 Guestfs_1_0_gir_FILES = $(patsubst %,$(srcdir)/%,$(introspection_sources))
 INTROSPECTION_GIRS += Guestfs-1.0.gir
 
