--- libguestfs-1.28.6/gobject/Makefile.am.orig	2015-03-01 21:20:01.977601510 +0100
+++ libguestfs-1.28.6/gobject/Makefile.am	2015-03-01 21:24:35.947590012 +0100
@@ -54,7 +54,7 @@
 libguestfs_gobject_1_0_la_CFLAGS = -I$(top_srcdir)/src -I$(srcdir)/include \
                                    -DGUESTFS_PRIVATE=1 \
                                    $(GOBJECT_CFLAGS)
-libguestfs_gobject_1_0_la_LDFLAGS = $(LDFLAGS) -L$(top_builddir)/src
+libguestfs_gobject_1_0_la_LDFLAGS = $(LDFLAGS)
 libguestfs_gobject_1_0_la_LIBADD = $(top_builddir)/src/libguestfs.la $(GOBJECT_LIBS) $(GIO_LIBS)
 
 pkgconfigdir = $(libdir)/pkgconfig
